coldfront/plugins/qumulo/utils/acl_allocations.py

```diff
@@ -231,13 +231,10 @@ def set_or_reset_allocation_acls(
 
         if is_base_allocation:
             aces.extend(AcesManager.default_copy())
-            aces.extend(AcesManager.everyone_ace)
             acl["aces"] = AcesManager.filter_duplicates(aces)
             qumulo_api.rc.fs.set_acl_v2(acl=acl, path=fs_path)
             aces.extend(AcesManager.get_allocation_aces(rw_groupname, ro_groupname))
-            acl["aces"] = AcesManager.filter_duplicates(
-                AcesManager.remove_everyone_aces(aces)
-            )
+            acl["aces"] = AcesManager.filter_duplicates(aces)
```

Comment:
@ris-bmulligan I want to confirm that calling filter_ducplicates in line 234 and again in line 237 is intentional.